runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -100,7 +100,8 @@ public Layout getCurrentLayout() {
                 } else if (re instanceof WrongEpochException) {
                     WrongEpochException we = (WrongEpochException) re;
                     log.warn("Got a wrong epoch exception, updating epoch to {} and "
-                            + "invalidate view", we.getCorrectEpoch());
+                            + "invalidate view {}", we.getCorrectEpoch());
+                    Sleep.sleepUninterruptibly(retryRate);
```

Comment:
Why is this sleep necessary? If we pick up a new layout it should no longer be in the wrong epoch, right? And that itself should be subject to network latency?

Reply:
Network latency is not sufficient to avoid thundering. The sleep doesn't do exponential back off anyways, so I'll remove it.